### PR TITLE
Lookup files

### PR DESCRIPTION
**Pull request by ShkarupaNick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,8 +125,10 @@ The following configuration fields are available:
     }
   }
 }
-
 ```
+</details>
+
+`type` field represents type of the file. You can find additional information about Unix file types [below](#ssh2-sftp-client-api-and-documentation-links);
```
Comment (shulkaolka) [inline README.md]:
```suggestion
**Note:** `type` field represents type of the file. You can find additional information about Unix file types [below](#ssh2-sftp-client-api-and-documentation-links);
```

File: README.md [hunk 2]
```diff
@@ -285,6 +292,220 @@ Default `No`. In case `No` is selected - an error will be thrown when object id
 ```
 </details>
 
+`type` field represents type of the file. You can find additional information about Unix file types [below](#ssh2-sftp-client-api-and-documentation-links);
```
Comment (shulkaolka) [inline README.md]:
```suggestion
**Note:** `type` field represents type of the file. You can find additional information about Unix file types [below](#ssh2-sftp-client-api-and-documentation-links);
```

File: README.md [hunk 3]
```diff
@@ -285,6 +292,220 @@ Default `No`. In case `No` is selected - an error will be thrown when object id
...
+}
+```
+
+The next component may read from `url` in `attachments` for a memory-efficient way to read/parse data. 
+
+#### List of Expected Config fields
+##### Behavior
+`Fetch All` - fetch all objects in one message in form of array, `Emit Individually` - emit each fetched object as separate message.
+##### Number of search terms
+Not required field, number of search terms. Determines the number of search terms that the entity must match. Need to be an integer value from 1 to 100. If this field is empty, action emits all entities with selected type.
```
Comment (stas-fomenko) [inline README.md]: integer value from the interval [0-99], not `value from 1 to 100`

File: component.json [hunk 4]
```diff
@@ -199,6 +199,39 @@
...
+          "required": true,
+          "model": {
+            "emitIndividually": "Emit Individually",
+            "fetchAll": "Fetch All"
+          },
+          "prompt": "Choose a Behavior type"
+        },
+        "numSearchTerms": {
+          "viewClass": "TextFieldView",
+          "label": "Number of search terms",
```
Comment (stas-fomenko) [inline component.json]: add a hint ("note"): "Enter an integer number from interval: [0-99]"

**Pull request merged**